two_points_on_the_same_branch.sh

```diff
@@ -239,9 +239,11 @@ for coq_opam_package in $sorted_coq_opam_packages; do
         # perform measurements for the NEW/OLD commit (provided by the user)
         if [ $RUNNER = "NEW" ]; then
             export OPAMROOT="$new_opam_root"
+            date
```

Comment:
Can you put a little bit of text with `echo` so the log is a bit easier to read?

Reply:
There is an echo right below, what do you want more?

Reply:
Ok then, if you want to do something a bit more fancy such as `echo "Testing new commig: $(date)"` , otherwise it is fine like this.

Reply:
Done.